### DISCUSSION TITLE
Sonarr test fails with 404 error and "Failed to get language profiles"

### DISCUSSION THREAD

**Discussion by jtermi03** [sct/overseerr#954] [Help] [answered]
When I attempt to test a Sonarr connection during initial setup, the UI returns a generic error. I am able to successfully add a Radarr connection. I am running TrueNAS 12 with Radarr, Sonarr, and Plex in jails and Overseerr in a docker container in an Ubuntu VM. Any ideas?

The logs show:

    2021-02-17T02:10:06.841Z [error][Sonarr API]: Request failed with status code 404
    2021-02-17T02:10:06.842Z [error][Sonarr]: Failed to get language profiles

Running develop-f1c335815f2f17465cdd36ceb223e78a58149b3b

Here's what I did, starting with a functional Plex/Radarr/Sonarr setup:

(Starting with a functional Plex/Radarr/Sonarr setup)
1. Create overseerr container using:
`sudo docker run -d -e LOG_LEVEL=info -e TZ=America/New_York -p 5055:5055 -v /srv/config/overseerr:/app/config --restart unless-stopped sctx/overseerr:develop`
2. Launch the wizard
3. Configure Plex, test, and save
4. Configure Radarr, test, and save
5. Configure Sonarr (see screenshot for configs)
6. Click Test

![image](https://user-images.githubusercontent.com/32025689/108148162-3f0ea480-709e-11eb-849d-8ac3ebadc86c.png)

**Comment by TheCatLady** [comment, marked as answer]
Hi @jtermi03, are you running Sonarr v3?  Language profiles are a new feature in v3, and as a result we only support Sonarr v3.

Also, be sure to make sure that if you have a base URL configured in Sonarr that it is set here as well!

**Comment by jtermi03** [comment]
Oh man, I didn't notice Sonarr v3 was a manual update, thanks!

On Tue, Feb 16, 2021 at 9:38 PM TheCatLady <notifications@github.com> wrote:

> Hi @jtermi03 <https://github.com/jtermi03>, are you running Sonarr v3?
> Language profiles are a new feature in v3, and as a result we only support
> Sonarr v3.
>
> Also, be sure to make sure that if you have a base URL configured in
> Sonarr that it is set here as well!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sct/overseerr/discussions/954#discussioncomment-375719>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHUKYWMF3TNZAWGTUW5TFADS7MT3VANCNFSM4XXQ4QEQ>
> .
>


-- 
joetermini@gmail.com

**Reply by soonic6** [reply]
In which version was the Language Profile for Sonarr added? I cant see this option on 1.19.1

**Reply by sct** [reply]
As said above, only version 3+ is supported.

**Reply by soonic6** [reply]
i know, but i found out the Language Profile is just available in sctx/overseerr:develop

**Reply by sct** [reply]
Ah sorry, I misunderstood. Yes. It will be in the next release.
